src/components/accessories/billTable/BillTable.tsx

```diff
@@ -20,37 +19,40 @@ import {
   getFromFields,
 } from "../../../libraries/formDataHandling/functions";
 import { get, has } from "lodash";
-import { computeBillSummary } from "./config";
-import { TUserCredentials } from "../../../state/main/types";
 import SelectField from "../selectField/SelectField";
 import "./styles.scss";
 import {
   Accordion,
   AccordionDetails,
   AccordionSummary,
 } from "@material-ui/core";
-import { FilterList, Person } from "@material-ui/icons";
+import { FilterList } from "@material-ui/icons";
+import PatientPicker from "../patientPicker/PatientPicker";
 
-export const BillTable: FC<IBillTableProps> = ({
-  fields,
-  handleSummaryChange,
-}) => {
+export const BillTable: FC<IBillTableProps> = ({ fields }) => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
-  const header = ["date", "patient", "balance", "status"];
+  const header = [
+    "id",
+    "patient",
+    "patId",
+    "date",
+    "status",
+    "lastPayment",
+    "amount",
+    "balance",
+  ];
```

Comment:
This is a lot to show inside a responsive table. I think that you could move some coulmns inside the row,

Reply:
Ok